kedro-datasets/setup.py

```diff
@@ -0,0 +1,144 @@
+from itertools import chain
```

Comment:
Looks good on a quick first inspection but would like to give a closer one on the second round.